### DISCUSSION TITLE
Get error on AIR SDK 50.2.2.2, when close the app windows with webview.

### DISCUSSION THREAD

**Discussion by sol-oka** [airsdk/Adobe-Runtime-Support#2553] [General]
[Fault] exception, information=ArgumentError: Error #2004: One of the parameters is invalid.

![image](https://user-images.githubusercontent.com/102789019/228153121-86a65e37-78c9-4652-b43e-e4f683a217c0.png)

**Comment by ajwfrost** [comment]
This is covered in #2532 and will be fixed in the next release of AIR SDK (still aiming for release at the end of this week i.e. 31 March)

**Reply by itlancer** [reply]
Fixed with latest AIR 50.2.2.3.
Thanks!
